### ISSUE TITLE
Seeking to zero freezes and silently closes app in iOS

### ISSUE THREAD

**Issue by mizutani256** [TheWidlarzGroup/react-native-video#2801]
# Bug

Calling `seek()` with value zero freezes the app then silently closes it. Using any other value (e.g. `1.0`) works.

Maybe related to #2556 ?

## Platform
iOS

## Environment info

React native info output:

```bash
info Fetching system and libraries information...
System:
    OS: macOS 12.3.1
    CPU: (4) x64 Intel(R) Core(TM) i5-7360U CPU @ 2.30GHz
    Memory: 29.54 MB / 8.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 16.15.1 - ~/.nvm/versions/node/v16.15.1/bin/node
    Yarn: 1.22.19 - ~/.nvm/versions/node/v16.15.1/bin/yarn
    npm: 8.11.0 - ~/.nvm/versions/node/v16.15.1/bin/npm
    Watchman: 2022.01.31.00 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.11.2 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 21.0.1, iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0
    Android SDK:
      API Levels: 31
      Build Tools: 31.0.0
      Android NDK: Not Found
  IDEs:
    Android Studio: Not Found
    Xcode: 13.1/13A1030d - /usr/bin/xcodebuild
  Languages:
    Java: Not Found
  npmPackages:
    @react-native-community/cli: Not Found
    react: 17.0.2 => 17.0.2 
    react-native: 0.68.2 => 0.68.2 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
```

Library version: 6.0.0-alpha.1

## Steps To Reproduce

1. Render a `<Video>` component
2. Call `seek(0.0)` using the video's reference
3. For comparison, repeat but call `seek(1.0)` instead

## Expected behaviour

1. Play the video normally

## Reproducible sample code

```tsx
// in the onLoad callback
videoRef.seek(0.0);
```

**Comment by mizutani256** [comment]
Environment info added

**Comment by ckaznable** [comment]
May have been fixed in 6.0.0-alpha.2? Reference #2743

**Comment by freeboub** [comment]
fixed on master
